docs/.vuepress/components/AlertDialogDoc.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Could you please run lint for this file to fix indentation and other styling issues.

Reply:
Fixed lint issues.